### PR TITLE
Clean up funding spaces text in header

### PR DESCRIPTION
**Pull request by bamader**
## Background
The funding space headers of each age group in the roster accordion are textually cluttered, making it hard to see at a glance what the filled status of each funding space is. This PR cleans up those headers to more succinctly display capacity information.

## GitHub Issue
https://github.com/ctoec/data-collection/issues/671

## Validation Plan
I've uploaded several different files featuring all the different funding spaces and have checked that names and counts get correctly displayed. Also tested that showing and hiding an accordion group hides the space totals info.

## Automated Testing
- [ ] All unit tests are passing.
- [ ] All e2e tests are passing.
- [ ] If applicable, unit tests have been added to cover this changeset. (not applicable)
- [ ] If applicable, e2e tests have been added to cover this changeset. (not applicable)

## Additional Context
Since the AC of the ticket asked for the space totals information to be hidden when the age group accordion is collapsed, I've moved the "header" portion into the content itself. This visually looks almost identical to me, but it seemed like the best way for the single Show/Hide button in the Accordion to control both the children roster and the space totals text.

### CODE REVIEW

**Comment by jlhogan** [comment]
LGTM; confirmed in my local. Make sure to get approval from @RyanHansz in QA before merge!

**Review by JordanGuinn — APPROVED** [review]

**Comment by bamader** [comment]
@jlhogan Do you want to approve after I QA with Ryan?

**Review by jlhogan — APPROVED** [review]

**Comment by jlhogan** [comment]
@bamader yes sorry, forgot to actually hit the submit on my review, so was just sitting here pending

File: client/src/containers/Roster/RosterSectionHeader.tsx [hunk 1]
```diff
@@ -1,55 +1,90 @@
...
   return (
-    <div className="display-flex flex-wrap">
-      {fundingSpaceCounts.map(({ fundingSpace, count }) => {
-        let displayStr = `${fundingSpace.source} ${fundingSpace.time} - ${count}`;
-        const _hideCapacity = hideCapacity || fundingSpace.capacity === -1;
-        displayStr += _hideCapacity
-          ? ` ${pluralize('space', count)} filled`
-          : `/${fundingSpace.capacity}`;
+    <div>
+      {fundingSpacesMap.map((st) => {
```
[the remark this answers is not in the document]
Comment (bamader) [reply client/src/containers/Roster/RosterSectionHeader.tsx:60]: An instance of one of the 'source-time' objects that helps hold the data accumulation. Which is not a particularly clear variable name when it's abridged, so I'll rename it for clarity!

**Comment by RyanHansz** [comment]
Looks much better!
Two small things:
- Looks like there's a horizontal scroll bar in the roster section when I look in chrome
- Can we extend the funding source column width enough so that School readiness doesn't break across two lines?

**Comment by bamader** [comment]
@RyanHansz Sorry, not seeing what you mean with school readiness in the screenshot. Do you have a different view just so I know which section you mean?

**Comment by bamader** [comment]
@RyanHansz Design tweaks addressed and deployed back to QA! Column has been expanded (this is the next size up though so if this is too big we have to go back down to the width where school readiness is split), scroll-bar taken care of, and I even adjusted the header padding so that the funding source names would line up vertically with where entries in the lower tables began (for that sweet sweet vertical alignment 😎 ).

**Comment by bamader** [comment]
@RyanHansz Okay this is back in QA now with all the latest changes. All tests have passed and we've got both approvals so once it's got the design OK, we'll be good to merge.

**Comment by RyanHansz** [comment]
Looks good to me!

**Pull request merged**